src/components/LoginForm/LoginForm.tsx

```diff
@@ -11,17 +11,22 @@ import InputLabel from '@mui/material/InputLabel';
 import OutlinedInput from '@mui/material/OutlinedInput';
```

Comment:
I think we could consider going back to the matter of merging `LoginForm` and `RegisterForm`. You already had sort of double work with introducing types to both of them separately. Types might get trickier when the 2 components become 1 but a lot of changes won't require introducing them to both the components separately. I'd rather you didn't do the merging as part of this task but let's make the separate task to do so high in priority.